enos/enos-scenario-autopilot.hcl

```diff
@@ -632,12 +632,50 @@ scenario "autopilot" {
     }
   }
 
+  step "verify_removed" {
```

Comment:
`verify_raft_node_removed`, otherwise I'm not sure what the step really implies.